Proposal-Process.md

```diff
@@ -0,0 +1,14 @@
+For a proposal to create or change a process foundation-wide, a PR should be
```

Comment:
`foundation-wide`: I originally struggled with what to put there and came up with `organization-wide` and `project-wide`. Neither of those felt *quite* right but I went with them on the theory of "just get *something* out there and let other people comment on it already".

Without noting my own misgivings about the terms, @mikeal flagged them as not accurate and (implicitly, perhaps) suggested `foundation-wide`. Here's Mikeal's' comment:

> The problem with "project" and "organization" wide is that it could be interpreted as "just Node.js" or "just the Node.js GitHub org" when what I believe we want to do is implement these "foundation-wide" so that they apply to any top level project we bring in.

(If you want more context: https://github.com/nodejs/inclusivity/pull/68#discussion_r48095093)

I'm happy to change the wording to something else, but I would rather be told what that wording is. I'm struggling coming up with good wording.

TL;DR: We've tried each of the following and none have proven satisfactory. What is the wording we should use instead?

* foundation-wide
* project-wide
* organization-wide